lib/galaxy/dependencies/conditional_requirements.txt

```diff
@@ -0,0 +1,10 @@
+# These dependencies are only required when certain config options are set
+psycopg2==2.6.1
+WebError==0.11
+python-openid
+MySQL-python
+fluent-logger
+raven
+pbs_python
+drmaa
+# PyRods not in PyPI
```

Comment:
Probably, but I guess we'll have to parse the entire `object_store_conf.xml` to find any configured S3 stores? The error message in `s3.py` is thankfully pretty helpful